Zilon.Core/Zilon.Core/Tactics/Spatial/ActorPathFindingContext.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections.Generic;
+
+using Zilon.Core.Graphs;
+using Zilon.Core.PathFinding;
+
+namespace Zilon.Core.Tactics.Spatial
+{
+    /// <summary>
+    /// Базовая реализация контекста поиска пути.
+    /// </summary>
+    public struct ActorPathFindingContext : IPathFindingContext
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement 'IEquatable<T>' in value type 'ActorPathFindingContext'.](https://app.codacy.com/app/kreghek/Zilon_Roguelike/pullRequest?prid=4592476)